src/f32/coresimd/vec3a.rs

```diff
@@ -26,7 +26,12 @@ pub const fn vec3a(x: f32, y: f32, z: f32) -> Vec3A {
 ///
 /// This type is 16 byte aligned.
 #[derive(Clone, Copy)]
+#[cfg_attr(
+    feature = "zerocopy",
+    derive(zerocopy::FromBytes, zerocopy::Immutable, zerocopy::KnownLayout)
+)]
 #[cfg_attr(feature = "bytemuck", derive(bytemuck::Pod, bytemuck::Zeroable))]
+#[cfg_attr(feature = "zerocopy", derive(zerocopy::IntoBytes))]
```

Comment:
Where the condition is the same it would be preferable for these to be in a single derive. Because the generated code gets formatted, you might be able to put template conditions in the `derive` scope to conditionally include or exclude things like `IntoBytes` and it might still end up formatted nicely in the output.